lib/Doctrine/DBAL/Migrations/Configuration/Configuration.php

```diff
@@ -782,18 +783,18 @@ public function createMigrationTable()
         }
 
         $this->connect();
-        if ($this->connection->getSchemaManager()->tablesExist([$this->migrationsTableName])) {
+        $schemaManager = $this->connection->getSchemaManager();
+        if ($schemaManager->tablesExist([$this->migrationsTableName])) {
             $this->migrationTableCreated = true;
 
             return false;
         }
 
-        $columns = [
-            $this->migrationsColumnName => new Column($this->migrationsColumnName, Type::getType('string'), ['length' => 255]),
-        ];
-        $table   = new Table($this->migrationsTableName, $columns);
+        $schema = new Schema([], [], $schemaManager->createSchemaConfig());
+        $table = $schema->createTable($this->migrationsTableName);
```

Comment:
The build is failling because there is a space missing before the equal to align it with the one from the line above.

Reply:
Fixed.